GETTING_STARTED.md

```diff
@@ -69,6 +69,11 @@ To login as the default user, use:
     email: ofn@example.com
     password: ofn123
 
+To sign up as the new user:
+- run `bundle exec sidekiq`
+- sign up a new user
+- click confirmation link auto-opened in your browser by sidekiq
```

Comment:
Yes, that works. But there's actually a better way. I think that we should change line 63:
```diff
-    bundle exec rails server
+    foreman start
```

That starts sidekiq and webpacker as well. Signing up a new user should be obvious after that. What do you think?